benchmark/torch/ppo/env_utils.py

```diff
@@ -102,7 +101,7 @@ def step(self, action_list):
                     next_obs = self.env_list[i].reset()
                 self.episode_steps_list[i] = 0
                 self.episode_reward_list[i] = 0
-                self.global_ob_rms = self.env_list[i].get_ob_rms()
+                self.eval_ob_rms = self.env_list[i].get_ob_rms()
```

Comment:
please try to replace by self.env_list[i].env.get_ob_rms() so that we can remove two api in LocalEnv and RemoteEnv，which is not that universal.

Reply:
done